.github/workflows/buildkit-daily.yml

```diff
@@ -0,0 +1,95 @@
+name: Publish buildkit daily container
+
+on:
+  schedule:
+    - cron: '0 17 * * *'
+  workflow_dispatch:
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: ${{github.repository_owner}}/aosc-os-buildkit-daily
+
+jobs:
+  tarball:
+    runs-on: [self-hosted, aosc-os, x64]
```

Comment:
I see the runner is tagged `AOSC-OS` instead of `aosc-os`, will it be a problem?